js/screens/CreateItem/styles.js

```diff
@@ -1,11 +1,36 @@
 import { StyleSheet, Dimensions } from 'react-native';
 import { fonts, colors } from '../../config/styles';
 const { height, width } = Dimensions.get('window');
+const heightimg = height / 4.27;
+const widthimg = width / 3.2;
 
 const styles = StyleSheet.create({
   container: {
     backgroundColor: 'white',
-    paddingHorizontal: 10,
+    flex: 1,
+    paddingTop: 10,
+    padding: 10,
+  },
+  selectedImage: {
+    width: widthimg,
+    height: heightimg,
```

Comment:
if you give flex property to an image it should work fine without giving it height and width. Make all the necessary changes in this file.

Reply:
hi sid, the Image tag should have height and width or it won't display an image